src/grt/src/GlobalRouter.tcl

```diff
@@ -160,15 +160,15 @@ sta::define_cmd_args "set_clock_routing" { [-clock_pdrev_fanout fanout] \
 }
 
 proc set_clock_routing { args } {
-  sta::parse_key_args "global_route" args \
+  sta::parse_key_args "set_clock_routing" args \
     keys { -clock_pdrev_fanout \
            -clock_topology_priority
          }
 
   if { [info exists keys(-clock_topology_priority) ] } {
     set priority $keys(-clock_topology_priority)
     sta::check_positive_float "-clock_topology_priority" $priority
-    grt::set_alpha $clock_topology_priority
+    grt::set_alpha $priority
```

Comment:
this makes the function name set_alpha look pretty inconsistent. if it is really specific to clocks it should say so.
alpha is not a priority.  it is a parameter that controls the topology of the routing tree (trading off path depth and total wire length). you documented that, right?

Reply:
I'm using "priority" to indicate what will be prioritized during the tree topology construction. The grt section in the readme explains it. Should it have better terminology?